llvm/test/CodeGen/AArch64/ldp-stp-control-features.ll

```diff
@@ -0,0 +1,390 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 3
+; RUN: llc < %s -O2 -mtriple=aarch64 -mcpu=ampere1 | FileCheck %s --check-prefixes=CHECK
+; RUN: llc < %s -O2 -mtriple=aarch64 -mcpu=ampere1a | FileCheck %s --check-prefixes=CHECK
+; RUN: llc < %s -O2 -mtriple=aarch64 | FileCheck %s --check-prefixes=CHECK-LABEL-DEFAULT,CHECK-DEFAULT,CHECK-NEXT-DEFAULT
```

Comment:
These should be --check-prefix=CHECK-DEFAULT, same for the ones below. The LABEL and NEXT prefixes are automatic.

They should generate check lines like `; CHECK-DEFAULT-NEXT:`, I believe.

Reply:
Oh ok, I thought this was done with the normal check only. Good to know from now on! Thanks!